### PR TITLE
Responsive Nav Menu Example issue #10

### PR DESCRIPTION
**Pull request by nelsonic**
While building the `start|stop timer` feature https://github.com/dwyl/app/issues/265, 
I tried to implement a mobile-first (_responsive_) navigation menu in Tachyons and **failed _miserably_**. 🤦‍♂ 
After some _googling_ I discovered issue #10 (_which I had opened a while back ..._) 
(_It's always good to re-discover a @dwyl issue with some helpful content! the system works!!_) 

@Cleop left a comment https://github.com/dwyl/learn-tachyons/issues/10#issuecomment-308490540 linking to the `dwyl-site` implementation of the CSS-only responsive navigation menu: https://github.com/dwyl/dwyl-site/pull/264/files#diff-b0638442d6af953ff33fca3d49113579L18 
The link and brief explanation is _much_ better than nothing, so thanks for the comment! 🥇 
And judging by how much ❤️ the comment has received from the community:
lots-of-love-for-cleops-comment
A _lot_ of people using Tachyons have found this useful. 🎉 

But ... rather than forcing people to read through a commit which isn't very beginner-friendly,
I've split out the code and simplified it into an example with in-line comments and explanation.

# This PR includes

+ [x] Screenshots and explanation in the `README.md`: https://github.com/dwyl/learn-tachyons/tree/responsive-nav-menu-issue%2310#responsive-navigation-menu
+ [x] [`examples/nav-menu.html`](https://github.com/dwyl/learn-tachyons/blob/responsive-nav-menu-issue%2310/examples/nav-menu.html) file people can learn from and play with.
 
@iteles / @SimonLab please take a look when you can. (_not urgent, just nice-to-have_) 
Thanks! 🌻

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -747,6 +748,97 @@ what the ideal dimensions are for the image they should upload.
   + `fw1` - "font weight 1", used to "fade" the font on the instructions.
   + `fw5` - "font weight 5", make the pixel dimensions more prominent.
 
+To view this in action,
+run
+`npm satart`
```
Comment (iteles) [inline README.md]:
```suggestion
`npm start`
```

File: README.md [hunk 2]
```diff
@@ -747,6 +748,97 @@ what the ideal dimensions are for the image they should upload.
...
+
+The _expanded_ burger menu:
+
+![mobile-view-burger-nav-menu](https://user-images.githubusercontent.com/194400/73554487-b2b62300-4443-11ea-981a-cf26e3818688.png)
+
+This CSS-only (_no `JavaScript` required!_) responsive navigation menu
+relies on [Isabel Castillo](https://github.com/isabelc)'s
+brilliant _hidden_ `checkbox` idea:
+https://isabelcastillo.com/pure-css-mobile-toggle-menu
+(shared by @iteles in
```
Comment (iteles) [inline README.md:778]: Haha I didn't need the shout-out but thank you!

File: README.md [hunk 3]
```diff
@@ -747,6 +748,97 @@ what the ideal dimensions are for the image they should upload.
...
+  /* when checkbox is checked, display menu (sibling element) */
+  #burger:checked ~ .menu {
+    max-height: 100%;
+  }
+</style>
+```
+
+Relevant to this quest is understanding the Tachyons display system:
+https://tachyons.io/docs/layout/display
+_Specifically_, the understand that
```
Comment (iteles) [inline README.md]:
```suggestion
_Specifically_, understanding that
```

**Review by iteles — APPROVED** [review]
🙌  Thanks @nelsonic. Corrected a couple of typos, ready to go.

File: README.md (hunk 2 above)
Comment (nelsonic) [reply README.md:778]: you found the pearl. definitely worth recognising. recognition is core to our _culture_. ❤️

**Review by SimonLab — APPROVED** [review]
menu tested and working well :+1:

**Pull request merged**